src/Data/Mapper/PositionMapper.php

```diff
@@ -27,4 +27,24 @@ public function getPositions($options = [])
 
         return [$this->mapObjects($result), $paginate];
     }
+
+    protected function mapObjects(array $records)
+    {
+        $results = [];
+        foreach ($records as $record) {
+            $entity = new $this->entity;
+            $entity->setId($record['id']);
+            $entity->setAccountId($record['account_id']);
+            $entity->setName($record['name']);
+            $entity->setColor($record['color']);
+            $entity->setSort($record['sort']);
+            $entity->setCreatedAt($record['created_at']);
+            $entity->setUpdatedAt($record['updated_at']);
+            $entity->setDeleted($record['is_deleted']);
```

Comment:
I agree, I like what you have talked about. I'll get it implemented and then we can take a look at it again.